wasm/jit/jit_amd64.go

```diff
@@ -1724,7 +1703,7 @@ func (c *amd64Compiler) compileDiv(o *wazeroir.OperationDiv) (err error) {
 // compileDivForInts emits the instructions to perform division on the top
 // two values of integer type on the stack and puts the quotient of the result
 // onto the stack. For example, stack [..., 10, 3] results in [..., 3] where
-// the remainder is discarded. See compileRem for how to acquire remainder, not quotient.
```

Comment:
Is this removal intended? If it's intended, it's ok, I just wanted to confirm

Reply:
intended as it's just redundant!